src/Documents.ts

```diff
@@ -32,8 +32,8 @@ export class Class implements Pointer.Library, Pointer.Validator, ObjectSchema.R
 	private _jsonldConverter:JSONLD.Converter.Class;
 	get jsonldConverter():JSONLD.Converter.Class { return this._jsonldConverter; }
 
-	private _documentDecorators:Map<string, {decorator:Function, parameters?:any[]}>;
-	get documentDecorators():Map<string, {decorator:Function, parameters?:any[]}> { return this._documentDecorators; }
+	private _documentDecorators:Map<string, {decorator:( object:Object, ...parameters:any[] ) => Object, parameters?:any[]}>;
```

Comment:
You are using the type `{decorator:( object:Object , ...parameters:any[] ) => Object, parameters?:any[]  }` everywhere. Please create a type for it to improve the readability.